### DISCUSSION TITLE
C# native language support

### DISCUSSION THREAD

**Discussion by Febbie15** [mitmproxy/mitmproxy#5391] [Q&A]
**C# native language support for BrowserUp MITM Proxy** will be very useful for us as per our project requirement. As alternate option, we planned to use REST API in our code , but when I click the document of REST API , it shows 404 (Page not available) error. Kindly help me out.

**Comment by mhils** [comment]
Please use https://browserup.com/ for support regarding browserup's mitmproxy fork.

**Reply by Febbie15** [reply]
ok thanks. We posted it in Browserup's page.

**Comment by ericbeland** [comment]
Sorry about spillover. I'll add something to the readme in our fork to try and help direct folks better.
